File: solutions/1683-maximum-number-of-coins-you-can-get/solution.py
class Solution(object):
    def maxCoins(self, piles):
        """
        :type piles: List[int]
        :rtype: int
        """
        piles=sorted(piles)[::-1]
        sum1=0
        for i in range(1,len(piles)-len(piles)/3,2):
            sum1+=piles[i]
        return sum1

File: solutions/1683-maximum-number-of-coins-you-can-get/solution.java
class Solution {
    public int maxCoins(int[] piles) {
        Arrays.sort(piles);
        int sum1=0,n=piles.length;
        for(int i=n/3;i<n;i+=2)
            sum1+=piles[i];
        return sum1;
    }
}
